cache-config/testing/docker/trafficserver/Dockerfile

```diff
@@ -63,10 +63,12 @@ RUN if [[ ${OS_VERSION%%.*} -ge 8 ]]; then \
 			curl \
 			gcc-toolset-9 \
 			gcc-toolset-9-runtime \
+			geoip \
 			jansson \
 			jansson-devel \
 			libmaxminddb \
-			libmaxminddb-devel); \
+			libmaxminddb-devel \
+			libuuid ); \
```

Comment:
`rockylinux:8`, `centos:7`, and `centos:8` all include the `libuuid` package. Is specifying it needed?

```shell
[zrhoffman@computer ~]$ for image in rockylinux:8 centos:{7,8}; do docker run --rm "$image" rpm -q libuuid; done
libuuid-2.32.1-35.el8.x86_64
libuuid-2.23.2-65.el7.x86_64
libuuid-2.32.1-27.el8.x86_64
```